numpy/distutils/mingw32ccompiler.py

```diff
@@ -15,7 +15,6 @@
 import textwrap
 
 # Overwrite certain distutils.ccompiler functions:
-import numpy.distutils.ccompiler  # noqa: F401
```

Comment:
It's possible this import is needed to do some monkey-patching. I don't know for sure.